api/XDS_PROTOCOL.md

```diff
@@ -285,6 +285,50 @@ admin:
 
 ```
 
+### Incremental xDS
+
+Incremental xDS is separate xDS endpoint available for ADS that allows:
+
+  * Incremental updates of the list of tracked resources by the xDS client.
+    This supports Envoy on-demand / lazily requesting additional resources. For
+    example, this may occur when a request corresponding to an unknown cluster
+    arrives.
+  * The xDS server can incremetally update the resources on the client.
+    This support the goal of scalability of xDS resources. Rather than deliver
+    all 100k clusters when a single cluster is modified, the management server
+    only needs to deliver the single cluster that changed.
+
+An xDS Incremental session is always in the context of a gRPC bidirectional
+stream. This allows the xDS server to keep track of the state of xDS clients
+connected to it. There is no REST version of Incremental xDS.
+
+In Incremental xDS the nonce field is required and used to pair
+IncrementalDiscoveryResponse to a IncrementalDiscoveryRequest ACK or NACK.
+Optionaly, a response message level system_version_info is present for
+debugging purposes only.
+
+IncrementalDiscoveryRequest can be sent in 3 situations:
+  1. Initial message in a xDS bidirectional gRPC stream.
+  2. As a ACK or NACK response to a previous IncrementalDiscoveryResponse.
+     In this case the response_nonce is set to the nonce value in the Response.
```

Comment:
Nit: use quote style for `response_nonce` and `error_detail` here.

Reply:
Done.